src/utils/draftutil.js

```diff
@@ -241,23 +248,17 @@ function createPacks(draft, format, seats, nextCardFn) {
 }
 
 // NOTE: format is an array with extra attributes, see getDraftFormat()
-export function populateDraft(draft, format, cards, bots, seats) {
+export function populateDraft(draft, format, cards, bots, seats, seed = false) {
   let nextCardFn = null;
 
   if (cards.length === 0) {
     throw new Error('Unable to create draft: no cards.');
   }
-  if (bots.length === 0) {
```

Comment:
If you remove these checks then they should be re-added to (before) the routes/calls that do require them.